pluginconfig/pluginconfig-311.yaml

```diff
@@ -56,6 +56,7 @@ versions:
       sync: quay.io/openshift-on-azure/azure:v7.0
       tlsProxy: quay.io/openshift-on-azure/azure:v7.0
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod07092019
+      metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.153-2
```

Comment:
don't update older versions - only the most recent

Reply:
you have to otherwise config validation fails.

Reply:
then fix the validation ;-) -- see how it was done in Leszek's PR